tests/virtual_machines_test.go

```diff
@@ -33,6 +36,24 @@ var _ = Describe("Virtual Machines", func() {
 	})
 
 	Context("Check the client", func() {
+		BeforeEach(func() {
```

Comment:
Would it make sense to have this check in the main BeforeAll?

Reply:
done